packages/components/src/Card/Card.tsx

```diff
@@ -70,7 +72,12 @@ export function Card({
     );
   } else if (url) {
     return (
-      <a className={className} href={url}>
+      <a
+        className={className}
+        target={external ? "_blank" : "_self"}
+        rel={external ? "noopener noreferrer" : "tag"}
+        href={url}
+      >
```

Comment:
```suggestion
      <a
        className={className}
        href={url}
        {...(external && { target: "_blank", rel: "noopener noreferrer" })}
      >
```

Could I suggest this? You shouldn't need to worry about the anchor when it's not external, and this way we apply the `target` and `rel` at the same time.

Reply:
Sure sounds good

Reply:
@joelwarrington-jobber So we don't want to pass target=_self in case of external=false as suggested by @darryltec earlier